protos/feast/core/DataSource.proto

```diff
@@ -209,9 +209,6 @@ message DataSource {
 
     // Snowflake schema name
     string database = 4;
-
-    // Snowflake warehouse name
-    string warehouse = 5;
```

Comment:
should be added as a `reserved` index up top. https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
done